bin/dfx-ckbtc-import

```diff
@@ -67,16 +67,12 @@ ckbtc_import() {
     local remote_name="$1"
     local local_name="$2"
     local local_path="$(c="$local_name" jq -re '.canisters[env.c].candid' dfx.json)"
```

Comment:
This has a performance impact.  Why not download to a location that has the version in the path, then copy to wherever you need it?  That too would guarantee that you have the desired version, even if the download is skipped.

Reply:
I guess my use case might be different than yours? I just want to create a snapshot occasionally, which is slow anyway and then only use the snapshot. So I have no need for the caching.
While I agree your suggestion would allow both caching and correct versions, it also increases complexity and effort.
If you think it's worth it, do you mind doing it?